docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -145,7 +145,7 @@ A JSON array containing the deleted rules.
 
 IMPORTANT: This API has been deprecated since version 8.2, and is scheduled for end of life in Q4 2023. Please use the <<bulk-actions-rules-api-action, bulk action API>> instead.
```

Comment:
Is this endpoint now "dead"? See it says end of life is 04/2023.

Reply:
@banderror @peluja1012 can you provide insight on this? Can we remove the [Bulk rule action](https://www.elastic.co/guide/en/security/master/bulk-actions-rules-api.html) API docs? If yes, should we remove it for 8.8-8.11, or just 8.11?

Reply:
@yctercero @nastasha-solomon 👋 

> Is this endpoint now "dead"? See it says end of life is 04/2023.

It's not dead. All the bulk rules CRUD endpoints are still functioning and haven't been deleted yet.

> Can we remove the [Bulk rule action](https://www.elastic.co/guide/en/security/master/bulk-actions-rules-api.html) API docs?

I wouldn't remove the docs until we delete the endpoints from the codebase and this change is released in one of the future minor versions.

We have this ticket https://github.com/elastic/kibana/issues/130963 that we should work on before making any decisions about deleting the endpoints. Since this is not a priority right now because of so much work that needs to be done for Serverless, @nastasha-solomon could we maybe update the docs to extend this end-of-life date? We could also specify only the year, such as `is scheduled for end of life in 2024`.

Reply:
Thanks for the additional information, @banderror! I can file a separate PR to change the end-of-life date since the update is technically outside of the scope of this PR. Should I also extend the end-of-life date for the following endpoints too:
- [Bulk delete](https://www.elastic.co/guide/en/security/master/bulk-actions-rules-api.html)
- [Bulk update](https://www.elastic.co/guide/en/security/master/bulk-actions-rules-api.html#bulk-actions-rules-api-update)

Reply:
@nastasha-solomon Sounds good to me, and yes, please update the other bulk CRUD endpoints in this file. Basically, all the endpoints in this file marked as `IMPORTANT: This API has been deprecated since version 8.2, and is scheduled for end of life in Q4 2023.` should be updated. Thank you!

Reply:
Resolving this in https://github.com/elastic/security-docs/pull/4164.